testing/test-scripts/helloworld-v1-v2-route-rules.yaml

```diff
@@ -24,3 +24,5 @@ spec:
       - weight: 0.25
         tags:
         - "version=v2"
+      - tags:
+        - "version=v1"
```

Comment:
Don't understand why additional tag needed here.
Wanting to have v2 with 25% probability. Needs priority fix as in #512.